docs/source/adaptive_learning.rst

```diff
@@ -45,16 +48,43 @@ If you prefer to work with the curl, you need to follow these two stages to uplo
   In this part, you should only copy your token (provided to you in step 0) and paste it instead of :code:`TOKEN` in the command. ::
 
       curl -X GET "https://api.neuralet.io/api/v1/file/upload/" -H  "accept: application/json" -H "Authorization: Bearer TOKEN"
-      
-  After running this command, it will return a json containing two items. The first key is :code:`name`, which has your unique id (:code:`UUID`), and you should keep it for the config step. The second key is :code:`upload_link` that you should use as :code:`UploadURL` in the next part.
+  
+  .. note::
+    This endpoint has 2 more optional parameters:
+
+    1. label: Label indicates that in the future you can distinct the uploads from each other (in current situation only distinction between uploads is their ids which can be difficult to read).
+    
+    2. video_names (comma-separated names): Video names included in zip file in comma-separated format (like :code:`video1.mp4,video2.mp4`).
+
+    If you intend to pass these 2 parameters your url will be like this: ::
+        
+        https://api.neuralet.io/api/v1/file/upload/?label=crowded-street&?video_names=elm_street.mp4,helm_street.mp4
+
+
+
+After running this command, it will return a json containing two items. The first key is :code:`name`, which has your unique id (:code:`UUID`), and you should keep it for the config step. The second key is :code:`upload_object` which contains 2 keys: :code:`url` and :code:`fields`. 
+
+You can see the result in section below: ::
+    
+    "upload_object":{
```

Comment:
does the user needs to save all of these fields for next steps? If so please mention it.